stdlib/3/asyncio/subprocess.pyi

```diff
@@ -1,9 +1,16 @@
+import sys
 from asyncio import events
 from asyncio import protocols
 from asyncio import streams
 from asyncio import transports
 from typing import Any, Optional, Text, Tuple, Union, IO
 
+if sys.version_info >= (3, 6):
+    from os import PathLike
+    _PathLike = PathLike[str]
```

Comment:
Doesn't this also accept `PathLike[bytes]`? After all it seems to accept raw `bytes` as well. (I didnt'check.)

Reply:
I suppose it does. What's the correct syntax for this? `PathLike[str, bytes]` or something else?

Reply:
OK, updated.
I also changed the `*args` from `Any`. I duplicated the `str, bytes` there for readability, and in case `_PathLike` get refactored in future, as they could be any string, rather than paths.